config/alphanet/config.json

```diff
@@ -1,637 +1,648 @@
+						"publicKey":
+							"d1c3a2cb254554971db289b917a665b5c547617d6fd20c2d6051bc5dfc805b34"
+					},
+					{
+						"encryptedPassphrase":
+							"iterations=1&salt=eae1d88c2802365cab74b4cabb553365&cipherText=c9145118e3d1d5f88541104b47a4c165af8505221890f1695b09e036e4deafb66e1d77f0d22ee3a99a99c809808007f1731dae3a6d4bbefe4a97e639dca188df3154d0304a3434a4c4187466&iv=9437c2fcef8d56e46f41f23f&tag=1395e253833de0cb2f8de8df2d1cd445&version=1",
+						"publicKey":
+							"47b9b07df72d38c19867c6a8c12429e6b8e4d2be48b27cd407da590c7a2af0dc"
+					},
+					{
+						"encryptedPassphrase":
+							"iterations=1&salt=cb866d88c4120fb9f4bb59710ca0b227&cipherText=04289b5ded6f83cbfdc3a75b1994547e2a328d61dc27596b2a19baf7c4005ca48eb5821374728c4e1ba9e3f3de11efae68bacdc546d0290873e2e7d8a466ef2638a13ad5502c5ffadbee286b6972&iv=513251d83c9e8b0561f0dfaa&tag=1de6aa64082a39e3e45b967d19394a6c&version=1",
+						"publicKey":
+							"9a7452495138cf7cf5a1564c3ef16b186dd8ab4f96423f160e22a3aec6eb614f"
+					},
+					{
+						"encryptedPassphrase":
+							"iterations=1&salt=9d8281f9bb778493bb23d994e94179ad&cipherText=6ac4c4386a7c2f706323dba997521b721c1a1ea4dbafd7381ce1f6c900b2dbc78b82c9bcfc5f698eb3c65384e70e10cd7dd206d8145b413fc222384ea626c5ec4642a0890f4dd3dfb9b109&iv=2b84ee759af48cdf02ae5907&tag=31eab344bdc7dedc3b9934d1f2068616&version=1",
+						"publicKey":
+							"c4dfedeb4f639f749e498a2307f1545ddd6bda62e5503ac1832b122c4a5aedf9"
+					},
+					{
+						"encryptedPassphrase":
+							"iterations=1&salt=6b36e67e7e17ef28ca6dcce6ec05b2e9&cipherText=c79a29e0f615751295db1481dde6cfb022f6256db332ba4ea1b98be2dbcdebc53a94a7e2e2f2d5f7a51c04dc7ef0807589a08ecb74c96add9a7ed45624e84d46e3dc6dfd34646233d4&iv=1593d256d8923854f45f4c91&tag=317bb0dc8d07adfbfa60301e86683b7f&version=1",
+						"publicKey":
+							"96c16a6251e1b9a8c918d5821a5aa8dfb9385607258338297221c5a226eca5c6"
+					},
+					{
+						"encryptedPassphrase":
+							"iterations=1&salt=cde0b7fbe70f0407e5386654ebfd03fe&cipherText=f1e571dfaa52ee53918474717cf3c16537a53aefb4731480098d150876cda0ba1e585ae5182bfdf374b8a3699c1b16dfa9e22adf4538b58c1afa7d5f36a03fc4de637872578f8fd5332713439dcc9f8985bc717cf08e200fa179&iv=6388afe81123047f07161608&tag=b20b64bee72f63e90e5070b1405703f0&version=1",
+						"publicKey":
+							"910da2a8e20f25ccbcb029fdcafd369b43d75e5bc4dc6d92352c29404acc350f"
+					},
+					{
+						"encryptedPassphrase":
+							"iterations=1&salt=937a488695507e8d4b5276ef4f612486&cipherText=95f0ec46a3305ba8e511d7e43b857ffb610a746c7edf1c551554d1c621da4b9354eee15cd2094ea9598f6865c6e2a7fe3b024d5470d6db2de7804d5cc4762b0ff4eeba04bc2eed66577220b7791b1c1cf38300&iv=05006351212b4b676ad9b157&tag=617cc0ff9dcb1fe31b298587f8461aaf&version=1",
+						"publicKey":
+							"eabfe7093ef2394deb1b84287f2ceb1b55fe638edc3358a28fc74f64b3498094"
+					},
+					{
+						"encryptedPassphrase":
+							"iterations=1&salt=d2d5903ed07b5909d955da
```

Comment:
sorry to jump in here without requesting, this port is invalid for `alphanet`

Reply:
Thanks for pointing it out, its fixed now.